packages/itwinui-css/src/code/codeblock.scss

```diff
@@ -51,7 +51,7 @@
         text-align: right;
         display: inline-block;
         padding: 0 $iui-s;
-        width: $iui-l;
+        width: calc($iui-l + 2 * $iui-s); // 2 * $iui-s for the padding
```

Comment:
I made this change specifically to prevent the test image from being changed.

Previously, `box-sizing: border-box` wasn't being applied on this element, so the padding was intrinsically being added the width, whereas now I'm manually adding it.